service/src/main/java/io/camunda/service/license/LicenseType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.license;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LicenseType {
+  SAAS("saas"),
+  PRODUCTION("production"),
+  UNKNOWN("unknown");
```

Comment:
Hmm it feels like these are not exclusive. Production doesn't imply Self-Managed to me. I don't feel so strongly about this and it's not a new thing introduced by this Enum, but the design feels a bit off to me. Also if my understanding of Unknown is correct, then it's not really a "type" of license, but rather an absence of a license

Reply:
Ok understood 👍 
Up to you, but perhaps constants are a better fit than an enum here then